docs/resources/json.md.erb

```diff
@@ -82,6 +82,24 @@ The `name` matcher tests the value of the filename as read from a JSON file vers
 
 <br>
 
+### Test JSON output from an HTTP API
+
+Our example API has a `/health` endpoint, which looks like this:
+
+    {
+      "service": {
+        "port": 3000,
+        "status": "ok"
+      }
+    }
+
+Example test:
```

Comment:
Just a thought, but it might be nice to link to the `rspec-its` docs, which provide detail on what to pass to `its`.